File: Domain/JobHistory.cs

namespace Domain
{
    public class JobHistory
    {
        public string Id { get; set; }
        public string JobName { get; set; }
        public string Position { get; set; }
        public DateTime Date { get; set; }
        public string Company { get; set; }
        public string Description { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;
        public bool IsUsed { get; set; } = true;

        public AppUser User { get; set; }
    }
}

File: client-app/src/models/JobHistory.ts
export interface JobHistory {
  id?: string;

  jobName: string;
  position: string;
  date: Date;
  company: string;
  description: string;
 
  user?: user;
  created?: Date;
  isUse?: boolean;
}

interface user {
  username: string;
  fullName: string;
  email: string;
  image: string | null;
}
